core/src/main/java/com/yahoo/ycsb/workloads/RestWorkload.java

```diff
@@ -0,0 +1,300 @@
+/**

```

Comment:
I agree with you that it would be nice to have RestWorkload merged into CoreWorkload but there are some concerns/points.
1. In order to benchmark REST-full web services we must already have URL endpoints to benchmark. This is why we have URL maps in RestWorkload. However the way core workload is written currently, it generates keys to benchmark instead of reading them from a collection of predefined keys. Which definitely makes sense for DBs but not for web services.
2. Considering the point above, if we still merge RestWorkload into CoreWorkload, it would require us to distinguish the key generation logic on the basis of some config value which would make the CoreWorkload code ugly and couple it with logic specific with a particular binding.
3. Delete Operations are not supported in CoreWorload, they need special handling for WebServices to benchmark.
4. Since CoreWorkload is the central component, updating it will require an exhaustive testing for all the other bindings. This is precisely what I wanted to avoid.
5. To minimize code duplication RestWorkload extends CoreWorkload. Hence most future updates in CoreWorkload will be propagated to RestWorkload automatically thereby reducing extra code efforts.

I understand the urge to have a single workload but given the different nature of WebServices there are some inherent differences from databases which makes this task a bit non-trivial. I am open to further improve this design in the future but for now, if we can get this pull request incorporated in the master then I will create another pull request with the improved design later.